static/app/views/issueDetails/groupSimilarIssues/similarStackTrace/item.tsx

```diff
@@ -161,16 +155,19 @@ class Item extends Component<Props, State> {
             const avgScore = aggregate?.[interfaceName];
             const scoreList = scoresByInterface?.[interfaceName] || [];
 
-            // If hasSimilarityEmbeddingsFeature is on, avgScore can be a string
-            let scoreValue = avgScore;
-            if (
-              (typeof avgScore !== 'string' && hasSimilarityEmbeddingsFeature) ||
-              !hasSimilarityEmbeddingsFeature
-            ) {
-              // Check for valid number (and not NaN)
-              scoreValue =
-                typeof avgScore === 'number' && !Number.isNaN(avgScore) ? avgScore : 0;
+            // Check for valid number (and not NaN)
+            let scoreValue =
+              typeof avgScore === 'number' && !Number.isNaN(avgScore) ? avgScore : 0;
+            // If hasSimilarityEmbeddingsFeature is on, map similarity score to score between 1-5
+            if (hasSimilarityEmbeddingsFeature) {
+              for (let i = 0; i <= similarityEmbeddingScoreValues.length; i++) {
+                if (scoreValue <= similarityEmbeddingScoreValues[i]) {
+                  scoreValue = i;
+                  break;
+                }
+              }
```

Comment:
scoreValue is being set to the index of the similarityEmbeddingScoreValues that is higher?

Reply:
maybe toss a comment there on that. translate 0-1 scale to 0-5 scale